app/src/main/java/ch/epfl/skysync/models/message/Message.kt

```diff
@@ -0,0 +1,10 @@
+package ch.epfl.skysync.models.message
+
+import ch.epfl.skysync.models.UNSET_ID
+import java.util.Date
+
+data class Message(
```

Comment:
Why does a message not have an owner ?

Reply:
Very good question, I have no idea. Will fix